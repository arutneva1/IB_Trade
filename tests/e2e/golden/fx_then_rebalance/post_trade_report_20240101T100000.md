| symbol | side | filled_shares | avg_price | notional | residual_drift_bps |
| --- | --- | --- | --- | --- | --- |
| AAA | BUY | 8.0000 | 101.00 | 808.00 | -800.00 |
