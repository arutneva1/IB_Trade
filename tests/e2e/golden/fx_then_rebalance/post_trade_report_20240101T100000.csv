symbol,side,filled_shares,avg_price,notional,residual_drift_bps
AAA,BUY,8.0,101.0,808.0,-800.0
